Xamarin.MacDev/MonoTouchSdk.cs

```diff
@@ -33,42 +33,42 @@ namespace Xamarin.MacDev
 {
 	public class MonoTouchSdk
 	{
-		static readonly IPhoneSdkVersion[] iOSSdkVersions = {
-			IPhoneSdkVersion.V6_0,
-			IPhoneSdkVersion.V6_1,
-			IPhoneSdkVersion.V7_0,
-			IPhoneSdkVersion.V7_1,
-			IPhoneSdkVersion.V8_0,
-			IPhoneSdkVersion.V8_1,
-			IPhoneSdkVersion.V8_2,
-			IPhoneSdkVersion.V8_3,
-			IPhoneSdkVersion.V8_4,
-			IPhoneSdkVersion.V9_0,
-			IPhoneSdkVersion.V9_1,
-			IPhoneSdkVersion.V9_2,
-			IPhoneSdkVersion.V9_3,
-			IPhoneSdkVersion.V10_0,
-			IPhoneSdkVersion.V10_1
+		static readonly AppleSdkVersion [] iOSSdkVersions = {
+			AppleSdkVersion.V6_0,
+			AppleSdkVersion.V6_1,
+			AppleSdkVersion.V7_0,
+			AppleSdkVersion.V7_1,
+			AppleSdkVersion.V8_0,
+			AppleSdkVersion.V8_1,
+			AppleSdkVersion.V8_2,
+			AppleSdkVersion.V8_3,
+			AppleSdkVersion.V8_4,
+			AppleSdkVersion.V9_0,
+			AppleSdkVersion.V9_1,
+			AppleSdkVersion.V9_2,
+			AppleSdkVersion.V9_3,
+			AppleSdkVersion.V10_0,
+			AppleSdkVersion.V10_1
```

Comment:
same comment, that seems incomplete
who's using this ?

Reply:
Same: VSMac.